e2e/utils/DebugDriver.ts

```diff
@@ -29,4 +31,25 @@ export class DebugDriver {
   getBalance(token: string) {
     return this.page.getByTestId(`balance-${token}`);
   }
+
+  async createStrategy(template: CreateStrategyTemplate) {
```

Comment:
maybe we should offer a text area in the debug page to paste strategy json - making this process much faster

Reply:
Ok, I'll do that. After measuring we can save 4sec/test :)

Reply:
For some reason filling the textarea instead of each field creates a insufficient-allowance error. I think the benefice (1.5sec/test) is not worth much more time trying to make it work.

Reply:
I created an issue to tackle this https://github.com/bancorprotocol/carbon-app/issues/968